lib/ansible/modules/utilities/logic/wait_for.py

```diff
@@ -82,6 +82,12 @@
       - Defaults to a multiline regex.
     type: str
     version_added: "1.4"
+  send_string:
+    description:
+      - String to send to socket
```

Comment:
Can you describe the usecase a bit more here? i had to read your PR to understand why i'd use this.

Reply:
I expanded the description.